sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/OciManifest.cs

```diff
@@ -8,5 +8,11 @@ namespace Azure.Containers.ContainerRegistry.Specialized
     [CodeGenModel("OCIManifest")]
     public partial class OciManifest
     {
+        /// <summary> Initializes a new instance of OciManifest. </summary>
+        public OciManifest()
+        {
+            Layers = new ChangeTrackingList<OciBlobDescriptor>();
+            SchemaVersion = 2;
```

Comment:
@lmolkova, I have moved setting the default value to the required value into the constructor so callers don't have to set it themselves, per the OCI spec: https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest-property-descriptions

Let me know if you have any concerns with this - if you agree, we can make sure all languages implement it this way.